### PR TITLE
rstudio-server: kill rserver when script exit

### PR DESCRIPTION
**Pull request by randy3k**
`rserver` process should be killed when script exit.

### CODE REVIEW

File: rstudio-server.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class RstudioServer < Formula
   homepage "http://www.rstudio.com"
+  head "https://github.com/rstudio/rstudio.git"
   url "https://github.com/rstudio/rstudio/archive/v0.99.484.tar.gz"
   sha256 "8ca4abccb9b554713077cf1057ac13abadfd7606f22ac3386b2a88a38ae8a427"
```
Comment (dpo) [inline rstudio-server.rb]: Please add a revision number.

Comment (randy3k) [inline rstudio-server.rb]: why do us fix to a revision?

Comment (dpo) [inline rstudio-server.rb]: A revision number is necessary so the bottles can be updated properly. Please add `revision 1` to the header.

Comment (randy3k) [inline rstudio-server.rb]: Updated.

I thought you wanted me to do something like
```
head "https://github.com/rstudio/rstudio.git", :revison => xxxxxx
```

**Comment by dpo** [comment]
Thank you!

**Pull request closed without merging**